ipatests/test_webui/ui_driver.py

```diff
@@ -732,8 +732,15 @@ def button_click(self, name, parent=None,
     def _button_click(self, selector, parent, name=''):
         btn = self.find(selector, By.CSS_SELECTOR, parent, strict=True)
         self.move_to_element_in_page(btn)
+
+        # The small timeout (up to 5 seconds) allows to prevent exceptions when
+        # driver attempts to click a button before it is rendered.
+        WebDriverWait(self.driver, 5, 0.2).until(
+            lambda d: btn.is_displayed(),
+            'Button is not displayed: %s' % name
```

Comment:
As the name can be empty, I suggest using selector instead

Reply:
Fixed.